library/network/test/susefirewall_proposal_test.rb

```diff
@@ -16,15 +36,11 @@
       allow(Yast::SuSEFirewallProposal).to receive(:ServiceEnabled).and_return(true)
     end
 
-    context "when iscsi is used" do
-      it "proposes opening iscsi-target firewall service and full firewall initialization on boot" do
-        allow(Yast::Linuxrc).to receive(:useiscsi).and_return("initial")
-
-        expect(Yast::SuSEFirewall).to receive(:full_init_on_boot).and_return(true)
-        expect(Yast::SuSEFirewall).to receive(:SetServicesForZones).with(["service:iscsitarget"], ["EXT"], true).and_return(true)
+    it "proposes opening iscsi-target firewall service and full firewall initialization on boot" do
+      expect(Yast::SuSEFirewall).to receive(:full_init_on_boot).and_return(true)
+      expect(Yast::SuSEFirewall).to receive(:SetServicesForZones).with(["service:iscsitarget"], ["EXT"], true).and_return(true)
```

Comment:
Yes and no, it tests what is called, not what is does with it. Besides, I'm just moving the current test.